src/main/java/picard/vcf/CollectVariantCallingMetrics.java

```diff
@@ -293,8 +312,16 @@ public void calculateDerivedFields() {
             super.calculateDerivedFields();
             // Divide by zero should be OK -- NaN should get propagated to metrics file.
```

Comment:
I didn't touch this code at all -- that's from a previous commit.  If you want it changed then let's have someone do it in a different PR.